index.js

```diff
@@ -11,11 +11,22 @@ var createSourceMapLocatorPreprocessor = function(args, logger, helper) {
     }
 
     function inlineMap(inlineData){
+
+      var charset = 'utf-8';
+      if (/^;charset=([^;]+);/.test(inlineData)) {
+        var matches = inlineData.match(/^;charset=([^;]+);/)
```

Comment:
Just extracted into topmost scope. `const` is only available on older `node`s with harmony flag